pyproject.toml

```diff
@@ -37,6 +37,7 @@ python = ">=3.7.1,<4.0"
 coloredlogs = "*"
 dill = "*"
 filelock = "*"
+importlib_metadata = "*"
```

Comment:
This one is out of curiosity, but I think there's a way to conditionally install based on Python version if I'm not mistaken?

Reply:
There is, but it's already there from some other dependency we have; happy to make it conditional if the dependencies can still override this.